packages/@aws-cdk/aws-codepipeline/lib/stage.ts

```diff
@@ -105,69 +90,100 @@ export class Stage extends cdk.Construct implements cpapi.IStage, cpapi.IInterna
     return this._actions.slice();
   }
 
-  public render(): CfnPipeline.StageDeclarationProperty {
+  public render(): any {
```

Comment:
I think stage rendering can move to pipeline. No real value in having a stage render itself.

Reply:
I left it in `Stage` to minimize changes, but since `Stage` is no longer part of the public API of the package, I think that's fine.